src/PackageManagement.VisualStudio/ScriptPackage.cs

```diff
@@ -49,26 +50,91 @@ public IEnumerable<IPackageAssemblyReference> AssemblyReferences
             }
         }
 
+        public IEnumerable<IScriptPackageFile> GetFiles()
+        {
+
+            if (_files == null)
+            {
+                _files = GetFilesCore();
+            }
+
+            return _files;
+
+        }
+
         private IEnumerable<IPackageAssemblyReference> GetAssemblyReferencesCore()
         {
             var result = new List<PackageAssemblyReference>();
+            var reader = GetPackageReader(_installPath);
+
+            if (reader != null)
+            {
+                var referenceItems = reader.GetReferenceItems();
+                // In 2.8.X, we get all assembly references by filtering all files' path
+                // here we get them by using GetNearest for any framework   
+                var files = NuGetFrameworkUtility.GetNearest<FrameworkSpecificGroup>(referenceItems,
+                                                                                     NuGetFramework.AnyFramework);
+                if (files != null)
+                {
+                    result = files.Items.Select(file => new PackageAssemblyReference(file)).ToList();
+                }
+            }
+          
+            return result;
+        }
+
+        private List<ScriptPackageFile> GetFilesCore()
+        {
+            var result = new List<ScriptPackageFile>();
+            var reader = GetPackageReader(_installPath);
+
+            if (reader != null)
+            {
+                result.AddRange(GetPackageFiles(reader.GetLibItems()));
+                result.AddRange(GetPackageFiles(reader.GetToolItems()));
+                result.AddRange(GetPackageFiles(reader.GetContentItems()));
+                result.AddRange(GetPackageFiles(reader.GetBuildItems()));
+                result.AddRange(reader.GetFiles().Where(path => IsUnknownPath(path)).Select(p => new ScriptPackageFile(p, NuGetFramework.AnyFramework)));
```

Comment:
it will get all files in the package, the last one get all unknow files, it's compatible with 2.8